packages/craco/lib/cra.js

```diff
@@ -38,16 +38,52 @@ function overrideModule(modulePath, newModule) {
 
 /************  Paths  *******************/
 
+let srcExp = /\/([a-zA-Z_\-]+)\/?$/;
```

Comment:
can't we just use `https://nodejs.org/docs/latest/api/path.html#path_path_sep` and `path` module methods instead of regex?

Reply:
I updated my regular expressions in my local repository to be directly compatible with the Windows platform.
It looks like this`let srcExp =/(\/|\\)?([a-zA-Z_\-]+)(\/|\\)?$/`

Reply:
Thanks for pointing that out, I've decided to drop the regular and use path.sep.